app/mod_api/controllers.py

```diff
@@ -85,6 +85,23 @@ def save_metadata(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
+@mod_api_blueprint.route("/package/<publisher>/<package>", methods=["DELETE"])
+def delete_data_package(publisher, package):
+    try:
```

Comment:
This is missing Doc-string for Flasgger. So I  am not able to manually test @subhankarb could you please fix this?